libraries/elm/src/WithDeclartaiveEvent.elm

```diff
@@ -0,0 +1,93 @@
+module WithDeclartaiveEvent exposing (main)
```

Comment:
```suggestion
module WithDeclarativeEvent exposing (main)
```

(and file rename)

Reply:
Thanks for pointing this out! I've updated it as a separate commit because the file name and import also needed to be updated.